src/Common/src/Interop/Unix/libssl/Interop.OpenSsl.cs

```diff
@@ -113,8 +113,13 @@ internal static bool DoSslHandshake(SafeSslHandle context, byte[] recvBuf, int r
 
         }
 
-        internal static int Encrypt(SafeSslHandle context, byte[] buffer, int offset, int count, int bufferCapacity, out libssl.SslErrorCode errorCode)
+        internal static int Encrypt(SafeSslHandle context, byte[] buffer, int offset, int count, out libssl.SslErrorCode errorCode)
         {
+            Debug.Assert(buffer != null);
```

Comment:
It makes it easier to debug right now on Unix, where we don't get line numbers as part of stack traces.  In a case like this where there are four asserts, we wouldn't know which one failed, making debugging a bit more challenging.  Hopefully once the portable PDB work lands and coreclr is able to take advantage of it, the messages will be less important.